content/synthetics/_index.md

```diff
@@ -2,19 +2,67 @@
 title: Synthetics
 kind: documentation
 beta: true
-description: Synthetics
+description: documentation
 aliases:
   - /integrations/synthetics/
 further_reading:
 - link: "https://www.datadoghq.com/blog/introducing-synthetic-monitoring/"
   tag: "Blog"
   text: "Introducing Datadog Synthetics"
+- link: "synthetics/browser_check"
+  tag: "Documentation"
+  text: "Configure a Browser Check"
+- link: "synthetics/uptime_check"
+  tag: "Documentation"
+  text: "Configure an Uptime Check"
 ---
 
 <div class="alert alert-warning">Synthetics is currently in beta. See the <a href="https://app.datadoghq.com/synthetics/beta">Datadog Synthetics Request form</a>.</div>
 
-Datadog is pleased to announce the upcoming availability of Synthetics, a whole new layer of visibility on the Datadog platform. By monitoring your applications and API endpoints via simulated user requests, Synthetics helps you ensure uptime, identify regional issues, and track application performance. By unifying Synthetics with your metrics, traces, and logs, Datadog allows you to observe how all your systems are performing as experienced by your users.
+## Overview
+
+Datadog Synthetics gives you a new layer of visibility on the Datadog platform. By monitoring your applications and API endpoints via simulated user requests and browser rendering, Synthetics helps you ensure uptime, identify regional issues, and track application performance. By unifying Synthetics with your metrics, traces, and logs, Datadog allows you to observe how all your systems are performing as experienced by your users.
+
```

Comment:
I'm just wondering if that makes sense to mention logs here since I don't think we have any plans to directly tie synthetics to logs? But I might be mistaken here, and it's true that we could go from APM to logs so in a sense that's true.

Reply:
This is the wording from the corporate website here, I think it's in the product roadmap

Reply:
it is actually tied to traces which are themselves tied to logs. So yeah I think the wording is fine.